src/app/directives/documentation.js

```diff
@@ -231,6 +232,29 @@
               .velocity('slideUp');
           }
         };
+
+        $scope.getBeautifiedExample = function (value) {
```

Comment:
Is there any way to extract this duplicated code? I noticed that `api-console.js` has an identical function.